pydicom/waveforms/numpy_handler.py

```diff
@@ -135,7 +135,7 @@ def generate_multiplex(
                 sensitivity = ch.get("ChannelSensitivity", 1.0)
                 correction = ch.get("ChannelSensitivityCorrectionFactor", 1.0)
                 arr[..., jj] = (
-                    (arr[..., jj] + baseline) * sensitivity * correction
+                    arr[..., jj] * sensitivity * correction  + baseline
```

Comment:
Sorry to nitpick, but I still see a warning here in the diff view for an extra space we might as well correct. 

@mrbean-bremen, can we make these show up inline in Conversation tab?  They don't show up in CI unless you click through to details.

Reply:
I don't know, but I don't think so, as this comes from the GitHub API, not from the action itself.
We could of course convert these warnings to errors, and let the build fail on errors - that would show it more clearly, though I don't like it too much.

Another possibility is to use [pre-commit](https://pypi.org/project/pre-commit) instead of `reviewdog` - this can be installed locally via pip and will install a pre-commit hook, that will prevent a commit on any error. There is also a related CI that uses the [pre-commit configuration](https://github.com/pytest-dev/pyfakefs/blob/main/.pre-commit-config.yaml) and will fail the build in case of errors, and also [create PRs](https://github.com/pytest-dev/pyfakefs/pull/743) for tool updates. I have been made aware of this recently and have used it for `black` and `flake8` (this doesn't really belong here - we may discuss this elsewhere).

Reply:
> this doesn't really belong here - we may discuss this elsewhere

Agreed!  Just wanted to point it out in case it was helpful to see the outputs.